iam-login-service/src/main/java/it/infn/mw/iam/api/tokens/RefreshTokensController.java

```diff
@@ -0,0 +1,115 @@
+package it.infn.mw.iam.api.tokens;
+
+import static it.infn.mw.iam.api.tokens.Constants.REFRESH_TOKENS_ENDPOINT;
+
+import it.infn.mw.iam.api.account.authority.ErrorDTO;
+import it.infn.mw.iam.api.tokens.exception.TokenNotFoundException;
+import it.infn.mw.iam.api.tokens.model.RefreshToken;
+import it.infn.mw.iam.api.tokens.model.TokensListResponse;
+import it.infn.mw.iam.api.tokens.service.TokenService;
+import it.infn.mw.iam.api.tokens.service.paging.TokensPageRequest;
+import it.infn.mw.iam.core.user.exception.IamAccountException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.converter.json.MappingJacksonValue;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Optional;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@RestController
+@Transactional
+@PreAuthorize("hasRole('ADMIN')")
+@RequestMapping(REFRESH_TOKENS_ENDPOINT)
+public class RefreshTokensController extends TokensControllerSupport {
+
+  public static final Logger log = LoggerFactory.getLogger(RefreshTokensController.class);
+
+  @Autowired
+  private TokenService<RefreshToken> tokenService;
+
+  @RequestMapping(method = RequestMethod.GET, produces = CONTENT_TYPE)
+  public MappingJacksonValue lisRefreshTokens(
+      @RequestParam(value = "count", required = false) Integer count,
```

Comment:
IIRC you don't have to specify the value for request param if the name of the request parameter is the same as the name of the java parameter, as is the case here, so you can save some space and have just the required parameter